src/course-unit/data/api.js

```diff
@@ -16,8 +16,8 @@ const getLmsBaseUrl = () => getConfig().LMS_BASE_URL;
 export const getCourseUnitApiUrl = (itemId) => `${getStudioBaseUrl()}/xblock/container/${itemId}`;
 export const postXBlockBaseApiUrl = () => `${getStudioBaseUrl()}/xblock/`;
 export const getXBlockBaseApiUrl = (itemId) => `${getStudioBaseUrl()}/xblock/${itemId}`;
+export const getXBlocksBaseApiUrl = () => `${getStudioBaseUrl()}/xblock/`;
```

Comment:
[important] We already have variable with this endpoint `export const postXBlockBaseApiUrl = ..`

Reply:
Removed, thanks